docs/upgrading/upgrading_v3.md

```diff
@@ -272,17 +272,27 @@ Last but not least, `retry.limit` is set to `0`. This is because `crawlee` has i
 
 For more info please refer to the Got [documentation](https://github.com/sindresorhus/got#documentation).
 
-### Removed options
+## Removed options
```

Comment:
i believe those should stay as they were. the whole block is about removal of requestAsBrowser, which is the second level heading

Reply:
This is how it looks now..., it was super broken before
![Google Chrome - 2022-07-08 at 15 05 39@2x](https://user-images.githubusercontent.com/17960496/177988867-4728dba7-0744-4628-8d33-ee22ff317549.png)

Reply:
OH I see what you meant. Woooops. Will fix